README.md

```diff
@@ -81,7 +81,7 @@ For the 'version' column, servers have "(each PR)" if their continuous integrati
 4) See [#136](https://github.com/solid/test-suite/issues/136)
 5) Due to architectural trade-offs, global locks are not supported in Inrupt ESS
 6) See [#137](https://github.com/solid/test-suite/issues/137)
-7) PSS supports PATCH with `application/sparql-update` but not with the newly required `text/n3`, see https://github.com/solid/solid-crud-tests/pull/53/files
+7) PSS and Solid-Nextcloud support PATCH with `application/sparql-update` but not with the newly required `text/n3`, see https://github.com/solid/solid-crud-tests/pull/53/files
```

Comment:
At the moment, none of the servers should pass the CRUD `PATCH` tests, since all https://github.com/solid/solid-crud-tests/pull/53/files does is fire empty patches, which MUST be rejected as per the spec. So any server passing here is broken by definition.